google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java

```diff
@@ -228,19 +227,20 @@ public AppEngineStandardRuntime getAppEngineStandardDeclaredRuntime(
 
   private Optional<String> getValueFromAppYaml(@NotNull String appYamlPathString,
       @NotNull String key) {
-    Path appYamlPath;
+    Yaml yamlParser = new Yaml();
```

Comment:
I don't know the API that well. More specifically, I don't know if the parser carries any state between successive calls. We're only calling it twice, so it should be fine.
If we end up going down the parsing into an AppEngineAppYamlFile object, we'll only need to call it once.